runners/core-construction-java/src/main/java/org/apache/beam/runners/core/construction/graph/ExecutableStage.java

```diff
@@ -144,12 +162,13 @@ default PTransform toPTransform() {
    * Return an {@link ExecutableStage} constructed from the provided {@link FunctionSpec}
    * representation.
    *
-   * <p>See {@link #toPTransform()} for how the payload is constructed. Note that the payload
-   * contains some information redundant with the {@link PTransform} due to runner implementations
-   * not having the full transform context at translation time, but rather access to an
-   * {@link org.apache.beam.sdk.runners.AppliedPTransform}.
+   * <p>See {@link #toPTransform(RunnerApi.Components)} for how the payload is constructed. Note
+   * that the payload contains some information redundant with the {@link PTransform} due to runner
+   * implementations not having the full transform context at translation time, but rather access to
+   * an {@link org.apache.beam.sdk.runners.AppliedPTransform}.
```

Comment:
This statement should not be true for long, but we still have reasons to desire `ExecutableStagePayload`s to be self-contained.

Reply:
Updated.